python/rascaline/splines.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+
+def generate_splines(
+    radial_basis,
+    radial_basis_derivatives,
+    max_radial,
+    max_angular,
+    cutoff_radius,
+    n_spline_points=101,
+):
+
```

Comment:
This needs a docstring, to be added to the documentation, and unit tests

Reply:
Is it good now?

Reply:
@Luthaf

Reply:
yep!